test/maps/plain.wmf/scripting/tribes/spritesheet_animation/init.lua

```diff
@@ -0,0 +1,90 @@
+dirname = "test/maps/plain.wmf/" .. path.dirname(__file__)
+
+tribes:new_productionsite_type {
+   msgctxt = "barbarians_building",
+   name = "barbarians_spritesheet_animation",
+   -- TRANSLATORS: This is a building name used in lists of buildings
+   descname = pgettext("barbarians_building", "Spritesheet Animation"),
+   helptext_script = dirname .. "helptexts.lua",
+   icon = dirname .. "menu.png",
+   size = "medium",
+
+   buildcost = {
+      log = 3,
+      granite = 1
+   },
+   return_on_dismantle = {
+      log = 1,
+      granite = 1
+   },
+
+   animations = {
+      unoccupied = {
+         pictures = path.list_files(dirname .. "unoccupied_??.png"),
+         hotspot = { 52, 64 },
+      }
+   },
+
+   indicate_workarea_overlaps = {
+      barbarians_spritesheet_animation = false
+   },
+
+   spritesheets = {
+      idle = {
+         directory = dirname,
+         basename = "idle",
```

Comment:
Please convert all the animation definitions to the new format: Get rid of `pictures`, `basename` and `directory`, add `animation_directory`. Same for the worker.